src/tests/ftest/util/server_utils.py

```diff
@@ -330,3 +506,18 @@ def kill_server(hosts):
     for host in hosts:
         subprocess.call(
             "ssh {0} \"{1}\"".format(host, '; '.join(kill_cmds)), shell=True)
+
+
+def clean_server(hosts):
+    """Clean the /mnt/daos dir and /tmp dir on the servers.
+
+    Args:
+        hosts (list): list of host names where servers are running
+    """
+    cleanup_cmds = ["if [ -e {0} ]; then umount {0}; fi".format("/mnt/daos"),
```

Comment:
As Brian correctly pointed out we should not have a test case umount/mnt `/mnt/daos`.  I think we should just remove this command.

Reply:
Will do.